### DISCUSSION TITLE
Automation: Send newsletter every X days

### DISCUSSION THREAD

**Discussion by X-Raym** [hizzle-co/noptin#545] [Ideas]
Hi,

Sending automated mails one per week is too much. Sending one per months is too slow.

So in addition to these avalaible options, a Send every X days would be very appreciated. we could for eg put 14 days for one newskeletter every two weeks.

Cheers !

**Comment by picocodes** [comment]
This has been added via https://github.com/hizzle-co/noptin/commit/1adcbed08d5a80731390cab52fa564bb6b8957f2

**Comment by X-Raym** [comment]
@picocodes wow that was quick ! excellent ! 🙏

**Comment by X-Raym** [comment]
@picocodes I just tested the update, nice and quick!
I think we just forgot to be able to set the Day, we can only set the hour.
I would like to be able to set on sunday

Thx!

**Comment by picocodes** [comment]
We'll add that in a future update, for now you need to create the automation on a sunday for days to start counting from the sunday.

**Comment by X-Raym** [comment]
@picocodes many thx for your consideration 🙏

**Reply by X-Raym** [reply]
ok so it means the value can be edited in the database if needed. But we are Sunday actually so I can do this way 🙂

**Comment by X-Raym** [comment]
@picocodes Hi ! I see this is closed, do we now have a UI to set the starting day? Thx!

**Comment by picocodes** [comment]
Yeah,

You can create a Periodic Email instead of a normal post digest campaign.

If you do this then use the  [[posts]] merge tag to display your posts.

https://noptin.com/guide/sending-emails/how-to-display-posts-in-email-campaigns/

Docs about the Periodic campaign type coming soon.
